### ISSUE TITLE
`isLessDetachedRulesetCall` reads `@dr·()` as a call, which Less reads as an at-rule that asks for its semicolon

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#724]
Found by the review of the branch closing [#359](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/359), which reads the name of such a call as Less does for its own question and leaves the older reading as it is.

### What minimal example or steps are needed to reproduce the bug?

```less
a {
	@dr$() // c
	;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "customSyntax": "postcss-less",
  "rules": {
    "@stylistic/less/declaration-block-trailing-semicolon": "never"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.less" --config myconfig.json --fix
```

### What did you expect to happen?

The semicolon reported and left where it stands, as behind any at-rule Less reads to its semicolon.

### What actually happened?

`--fix` takes the semicolon away, and Less refuses the output with `missing block or ending semi-colon`.

### What is the cause?

`isLessDetachedRulesetCall` tells a call by an empty `raws.afterName` and params opening on `()`, whatever the name. Less calls a detached ruleset only by `@` and a run of ASCII word characters and hyphens; a name holding any other character, `$`, `!`, `*`, `·` or `Ä`, is an at-rule to it, and `requiresTrailingSemicolon` answers such an at-rule as a call. No detached ruleset can be named that way, so the harm is confined to files spelling such a name.

Measured against `less@4.9.1` and `postcss-less@6.0.0`.
